viewing_party/party.py

```diff
@@ -1,23 +1,177 @@
+import json
+from textwrap import indent
+
 # ------------- WAVE 1 --------------------
 
+# Instantiate movie as dictionary if all fields are populated
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:

```

Comment:
Nice check for truthy-ness! Great intuition that if they're not all truthy, omitting a `return` will effectively return `None`.